Sources/Sentry/Profiling/SentryProfilerState.mm

```diff
@@ -28,7 +28,7 @@
                                  options:0
                                    error:nil];
     });
-    const auto symbolNSStr = [NSString stringWithUTF8String:symbol];
+    const auto symbolNSStr = [NSString stringWithUTF8String:symbol] ?: @"<nil>";
```

Comment:
I am uncertain if we should prefer an empty string or `<nil>` to indicate that the symbol could not be converted